Assets/Scripts/MaterialReader.cs

```diff
@@ -864,6 +866,21 @@ public void ChangeWindowEmissionColor(Material material, WindowStyle windowStyle
             }
         }
 
+        /// <summary>
+        /// Removes from cache all materials that have not been accessed from the time in minutes defined
+        /// by <see cref="AssetCacheThreshold"/>. If equals to <c>0</c> assets are never removed from cache.
+        /// </summary>
+        internal void ValidateCache()
+        {
+            if (AssetCacheThreshold == 0)
+                return;
+
+            float time = Time.realtimeSinceStartup;
```

Comment:
For a general purpose cache in a general purpose program, I would have used a virtual clock, that only progresses when the program itself makes progress; So that for machines that are, say, slow to load resources, the cache pruning is not also more aggressive (relatively speaking).
I'm not sure it would make any difference in the case of a game, if the program is slowed down by minutes there's probably other more important issues...

Reply:
The purpose of removing assets from cache here is to free memory (especially vram); I don't think game performance should affect this behaviour. If you have low memory you want the cache not to take too much space, even if it means longer loading times. If you have more memory than what is normally used by the game, you can increse the threshold.

Reply:
The problem I was thinking about is loaded assets that you benefit for less effective time, because they (or other assets) took time to load; That feels kinda silly

Reply:
Now i get what you mean. I don't think this is really an issue here  but i appreciate that you pointed that out.